src/ui/src/builder/BuilderTemplateInput.vue

```diff
@@ -186,7 +195,7 @@ const showAutocomplete = () => {
 const handleBlur = () => {
 	setTimeout(() => {
 		autocompleteOptions.value = [];
-	}, 100);
+	}, 300);
```

Comment:
I encountered cases where the handler reaction was too fast. We may have to change this behavior in the future to have a flag to show or hide the autocompletion.